gcp/api/integration_tests.py

```diff
@@ -479,8 +483,8 @@ def test_query_package(self):
         _api() + '/v1/query',
         data=json.dumps({
             'package': {
-                'ecosystem': 'Maven',
-                'name': 'org.apache.tomcat:tomcat',
+                'ecosystem': 'PyPI',
```

Comment:
Are there any tests we could add that actually trigger the pagination? 

Also curious about the reason for these query changes too.

Reply:
The previous paging implementation limited the pages for package queries to 16. Since this upped it to 500 the Maven query wasn't returning enough results to require showing another page.